.github/workflows/CI.yml

```diff
@@ -45,8 +45,7 @@ jobs:
         run: bundle exec rake spec
 
       - name: Running the Dangerfile for this repo
-        if: runner.os != 'Windows'
-        run: |
-          TOKEN='7469b4e94ce21b43e3ab7a'
-          TOKEN+='79960c12a1e067f2ec'
-          DANGER_GITHUB_API_TOKEN=$TOKEN RUNNING_IN_ACTIONS=true echo 'bundle exec danger --verbose'
+        env:
+          DANGER_GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
secrets cannot be accessed by forked PRs, so this change isn't doing what you want it to do I think